sources/PanelButton.cpp

```diff
@@ -68,6 +73,14 @@ PanelButton::CreatePanel(int8 which, const BView* target, uint32 flavors,
 		panel = new BFilePanel(B_OPEN_PANEL, &msgr, NULL, flavors, false, NULL,
 			filter);
 
-		panel->Window()->SetTitle(pairs[which].locale);
+		BString title = B_TRANSLATE("Filer");
+		title += ": ";
+
+		if (prefix != NULL)
+			title << prefix << ' ';
```

Comment:
For some reason, there are two spaces after the prefix. I don't quite understand why, bur removing the `<< ' '` here fixes that...
Also, any idea why Filer's file panels don't close with ESCAPE as it's usual?